### ISSUE TITLE
Android New Users Incorrect.

### ISSUE THREAD

**Issue by fatkid9** [googleanalytics/google-analytics-plugin-for-unity#150]
Android new users incorrect. Google Play shows 2.731 new users for 18 March but Google Analytics shows 4.325 new users. 
It is working fine for IOS but It's not working correct for Android. 

Any help ?

**Comment by tuncertirnavali** [comment]
No, there are also wrong numbers(active users, new users etc) for iOS. I can recommend Mixpanel, it works better for me.
